src/screens/Signup.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import { useState } from 'react';
+
+export default function SignUpForm() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [verifyPassword, setVerifyPassword] = useState('');
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>Hi there. Please create an account!</Text>
+      <View style={styles.rowContainer}>
+        <Text style={styles.label}>Email</Text>
+        <TextInput
+          style={styles.input}
+          value={email}
+          onChangeText={text => setEmail(text)}
+        />
+      </View>
+      <View style={styles.rowContainer}>
+        <Text style={styles.label}>Password</Text>
+        <TextInput
+          style={styles.input}
+          value={password}
+          onChangeText={text => setPassword(text)}
+        />
+      </View>
+      <View style={styles.rowContainer}>
+        <Text style={styles.label}>Verify Password</Text>
+        <TextInput
+          style={styles.input}
+          value={verifyPassword}
+          onChangeText={text => setVerifyPassword(text)}
+        />
+      </View>
+      <Text style={styles.text}>Password must include</Text>
+      <TouchableOpacity style={styles.button}>
+        <Text>Sign up</Text>
+      </TouchableOpacity>
+      <Text>Already have an account? Log in.</Text>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
```

Comment:
I saw that you created a styles.css file – using `styled-components` and keeping all styles in a separate file helps us stay organized! See the [styling guide ](https://www.notion.so/calblueprint/V4V-Styling-Guide-7382718aba314f9994000d51818c8110) for more info! (I'd also be happy to help with frontend stuff feel free to send a message anytime)